cmd/utils.go

```diff
@@ -52,6 +52,7 @@ type ProjectConfig struct {
 	Description     string
 	License         string
 	Maintainers     []Maintainer
+	Operator        string
```

Comment:
Is this used anywhere?

Reply:
Oh, I had the idea to add an option for users to specify which operator they want to deploy with. I think that is too much out of scope, so will remove it.